source/blog/2018-11-23-the-emberjs-times-issue-74.md

```diff
@@ -51,13 +53,9 @@ There are [detailed designs](https://github.com/runspired/rfcs/blob/ember-data-p
 
 ---
 
-## [SECTION TITLE](#section-url)
-
-
----
-
-## [SECTION TITLE](#section-url)
+## [Use VSCode? Help Wanted 🚧](https://github.com/Microsoft/vscode-docs/issues/1509#issuecomment-440084989)
 
+VSCode recently published [Using Ember in Visual Studio Code](https://code.visualstudio.com/docs/nodejs/emberjs-tutorial) on their docs site! Ember **enthusiasts** are encouraged to [submit PRs](https://github.com/Microsoft/vscode-docs/issues/1509#issuecomment-440084989) to improve the tutorial, particularly to the [client-side debugging story](https://github.com/Microsoft/vscode-docs/blob/master/docs/nodejs/emberjs-tutorial.md#common-questions).
 
```

Comment:
`Looking for help to get started? Drop by [#dev-ember-learning](https://discordapp.com/channels/480462759797063690/480777444203429888) on the [Ember Community Chat](https://discordapp.com/invite/zT3asNS) and ask away!`

Reply:
I want to use this fancy Github suggestion feature here, but I don't know how unfortunately 😿

Reply:
Here we go:
```suggestion

Looking for help to get started? Drop by [#dev-ember-learning](https://discordapp.com/channels/480462759797063690/480777444203429888) on the [Ember Community Chat](https://discordapp.com/invite/zT3asNS) and ask away!
```